spec/javascripts/unit/modules/track-start-page-tabs.spec.js

```diff
@@ -0,0 +1,29 @@
+describe('Track start page tabs module', function () {
+  'use strict'
+
+  beforeEach(function () {
+    GOVUK.analytics = { trackEvent: function () {} }
+  })
+
+  it('tracks submit events', function () {
```

Comment:
Is this test name correct? I thought this was tracking clicks on tabs and form submissions were separate.

Reply:
Oops, I think that's a copy and paste error from how I got the file started 🤦‍♂️  fixed now.